fsm/fsm_models.go

```diff
@@ -298,6 +298,10 @@ func (f *FSMContext) Correlator() *EventCorrelator {
 	return f.eventCorrelator
 }
 
+func (f *FSMContext) Attempts(h *swf.HistoryEvent) int {
```

Comment:
I would add a line comment here so users know the intended functionality of this call.